chipsec/utilcmd/mmio_cmd.py

```diff
@@ -45,7 +45,7 @@ class MMIOCommand(BaseCommand):
     >>> chipsec_util mmio write <MMIO_BAR_name> <offset> <width> <value>
 
     Examples:
-    
+
```

Comment:
Missed in the trailing white space one?

Reply:
Must not have been within the other range of errors?